subsys/bluetooth/host/audio/chan.c

```diff
@@ -1598,6 +1549,65 @@ int bt_audio_broadcast_source_stop(struct bt_audio_broadcast_source *source)
 	return 0;
 }
 
+int bt_audio_broadcast_source_delete(struct bt_audio_broadcast_source *source)
+{
+	struct bt_audio_chan *chan;
+	struct bt_audio_chan *tmp;
+	struct bt_le_ext_adv *adv;
+	int err;
+
+	CHECKIF(source == NULL) {
+		BT_DBG("source is NULL");
+		return -EINVAL;
+	}
+
+	chan = source->chan;
+
+	if (chan->state != BT_AUDIO_CHAN_CONFIGURED) {
```

Comment:
In the stop() command, the state is supposed to be STREAMING.
Here, it is supposed to the CONFIGURED.
Where/when does the change happen?

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/38353/files#r705257593